libs/librepcb/common/graphics/graphicslayer.cpp

```diff
@@ -214,15 +214,15 @@ void GraphicsLayer::getDefaultValues(const QString& name, QString& nameTr,
     // clang-format off
     // schematic
     h.insert(sSchematicReferences,      {tr("References"),            QColor(0, 0, 0, 50),        QColor(0, 0, 0, 80),        true});
-    h.insert(sSchematicSheetFrames,     {tr("Sheet Frames"),          Qt::black,                  Qt::darkGray,               true});
+    h.insert(sSchematicSheetFrames,     {tr("Sheet Frames"),          QColor("#96E0E0E0"),        QColor("#FFFFFFFF"),        true});
```

Comment:
Are you sure you want to modify the **schematic** layer? :wink:

Reply:
Whoopsie.